roles/networking_mapper/templates/network-config-values.j2

```diff
@@ -0,0 +1,103 @@
+---
+{% set ns = namespace(interfaces={}, ocp_index=0) %}
+data:
+{% for host in _network_mapper.instances.keys() -%}
+{%   for network in _network_mapper.instances[host]['networks'].values() -%}
+{%     set ns.interfaces = ns.interfaces | combine({network.network_name: network.interface_name}, recursive=true) -%}
+{%   endfor -%}
+{%   if host is match('^(ocp|crc).*') %}
+  node_{{ ns.ocp_index }}:
+{%     set ns.ocp_index = ns.ocp_index+1 %}
+    name: {{ _network_mapper.instances[host]['name'] }}
+{%     for network in _network_mapper.instances[host]['networks'].values() %}
+    {{ network.network_name }}_ip: {{ network.ip_v4 }}
+{%     endfor %}
+{%   endif %}
+{% endfor %}
+
+{% for network in _network_mapper.networks.values() %}
+  {{ network.network_name }}:
+    dnsDomain: {{ network.search_domain }}
+{%  if network.tools is defined and network.tools.keys() | length > 0 %}
+    subnets:
+{%    for tool in network.tools.keys() %}
+      - allocationRanges:
+{%      for range in network.tools[tool].ipv4_ranges %}
+        - end: {{ range.end }}
+          start: {{ range.start }}
+{%      endfor %}
+        cidr: {{ network.network_v4 }}
+{%        if network.gw_v4 is defined %}
+        gateway: {{ network.gw_v4 }}
+{%      endif %}
+        name: {{ tool }}
+{%    endfor %}
+{%  endif %}
+    prefix-length: {{ network.network_v4 | ansible.utils.ipaddr('prefix') }}
+    mtu: {{ network.mtu | default(1500) }}
+{%  if network.vlan_id is defined  %}
+    vlan: {{ network.vlan_id }}
+    iface: {{ network.network_name }}
+    base_iface: {{ ns.interfaces[network.network_name] }}
+{%  else %}
+    iface: {{ ns.interfaces[network.network_name] }}
+{%  endif %}
+    lb_addresses:
+      - {{ network.network_v4 | ansible.utils.ipmath(80) }}-{{ network.network_v4 | ansible.utils.ipmath(90) }}
```

Comment:
Nope, for this purpose we have the tools section in the mapper, with a range for it. Same goes for the NNCPs, there is no need to hardcode this here.

Reply:
hm care to point to it? Is it in the "tools" maybe?